tests/lang/s01_basics/s00_atoms/t01_statement.nim

```diff
@@ -58,47 +81,60 @@ block noreturn_statement:
 
     block break_for_loop:
       ## Break in the `for` loop operates identically to the `while` case - when
-      ## `break` is reached loop body execution is stopped. For more details see
-      ## specification on iterators.
+      ## `break` is reached, the loop 'body' or 'path' of execution is stopped. For more details see
+      ## specification on `iterators`.
 
       var value = 0
       for i in 0 .. 10:
+        ## Break in the `for` loop terminates it's execution.
         break
         value = 2
 
       doAssert value == 0
 
   block continue_statement:
     ## `continue` statement can be used in the `while` and `for` loops to skip
-    ## execution of the current loop.
+    ## execution of the current loop. It effectively ends the current cyclic path
+    ## of execution and returns to the beginning of the cyclic path. This is different
+    ## to `break` which ends the cyclic path altogether and instead returns to
+    ## its parent path.
 
     block continue_in_while:
       var value = 0
       var cond = true
 
-      while cond:
+      while cond: # This cyclic path can only be entered if `cond` is true.
```

Comment:
Move to documentation comment, this should be accessible for the current code parser if we ever decided to generate `manual.rst` from this specification.